libs/island-ui/core/src/lib/ToggleSwitch/ToggleSwitch.treat.ts

```diff
@@ -0,0 +1,121 @@
+import { style } from 'treat'
+import { theme } from '@island.is/island-ui/theme'
+const { spacing, color, typography } = theme
+
+export const toggleSwitchLarge = style({})
+// export const right = style({})
+export const toggleSwitchChecked = style({})
+export const toggleSwitchDisabled = style({})
+export const toggleSwitchHiddenLabel = style({})
+
+export const checkbox = style({})
```

Comment:
is this being used anywhere?

Reply:
It used to, but not anymore. Good catch.